homeassistant/components/reolink/config_flow.py

```diff
@@ -66,11 +66,16 @@ async def async_step_user(
     ) -> FlowResult:
         """Handle the initial step."""
         errors = {}
-        placeholders = {}
+        placeholders = {"error": ""}
 
         if user_input is not None:
             try:
                 host = await async_obtain_host_settings(self.hass, user_input)
+                if not host.api.is_admin:
+                    errors[CONF_USERNAME] = "not_admin"
+                    placeholders[
+                        "error"
+                    ] = f"User '{host.api.username}' has authorisation level '{host.api.user_level}', admin required"
```

Comment:
Putting this much non-localized text into the description via the placeholder defeats the purpose of the localized strings. I'm not a config flow expert but don't most integrations report this purely via the errors dictionary?

Reply:
You're correct @joncar 👍

Reply:
Alright, I have fixed this, the problem was that I was using '{username}' which escaped the { character.
The fix was to use ''{username}'' with 2x '.
Now I am using the description placeholder of only the username and password in the error message.